functions/Get-DbaDatabase.ps1

```diff
@@ -314,14 +314,30 @@ function Get-DbaDatabase {
                 if ($NoFullBackupSince) {
                     $lastFullBackups = $lastFullBackups | Where-Object End -gt $NoFullBackupSince
                 }
-                $inputObject = $inputObject | Where-Object { $_.Name -cnotin $lastFullBackups.Database -and $_.Name -ne 'tempdb' }
+                $dbComparedIn = @()
+                foreach ($ref in $inputObject.Name) {
+                    foreach ($dif in $lastFullBackups.Database) {
+                        if ($server.getStringComparer($server.Collation).Compare($ref, $dif) -eq 0 ) {
+                            $dbComparedIn += $ref
+                        }
+                    }
+                }
```

Comment:
```suggestion
                $dbComparedIn = @()
                foreach ($ref in $inputObject.Name) {
                    foreach ($dif in $lastFullBackups.Database) {
                        if ($server.getStringComparer($server.Collation).Compare($ref, $dif) -eq 0 ) {
                            $dbComparedIn += $ref
                        }
                    }
                }
                $copyOnlyComparedIn = @()
                foreach ($ref in $inputObject.Name) {
                    foreach ($dif in $lastCopyOnlyBackups.Database) {
                        if ($server.getStringComparer($server.Collation).Compare($ref, $dif) -eq 0 ) {
                            $dbComparedIn += $ref
                        }
                    }
                }
```

Reply:
BackupStatus was never getting set, All of the DBs with only Copy Only backups were filtered out.